src/TestAbode/TestAbode.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
To be consistent with the names of other projects, lets rename this to Abode.Tests

Reply:
Renamed. Thank you.